includes/Modules/Analytics_4.php

```diff
@@ -442,6 +451,38 @@ protected function create_data_request( Data_Request $data ) {
 				return function() use ( $batch_request ) {
 					return $batch_request->execute();
 				};
+			case 'GET:container-lookup':
+				if ( ! isset( $data['destinatonID'] ) ) {
+					return new WP_Error(
+						'missing_required_param',
+						/* translators: %s: Missing parameter name */
+						sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'destinatonID' ),
+						array( 'status' => 400 )
+					);
+				}
+
+				return $this->get_tagmanager_service()->accounts_containers->lookup( array( 'destinationId' => $data['destinatonID'] ) );
+			case 'GET:container-destinations':
+				if ( ! isset( $data['accountID'] ) ) {
+					return new WP_Error(
+						'missing_required_param',
+						/* translators: %s: Missing parameter name */
+						sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'accountID' ),
+						array( 'status' => 400 )
+					);
+				}
+				if ( ! isset( $data['containerID'] ) ) {
+					return new WP_Error(
+						'missing_required_param',
+						/* translators: %s: Missing parameter name */
+						sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'containerID' ),
+						array( 'status' => 400 )
+					);
+				}
+
+				return $this->get_tagmanager_service()->accounts_containers_destinations->listAccountsContainersDestinations(
+					"accounts/{$data['accountID']}/containers/{$data['containerID']}"
+				);
```

Comment:
`container_id` is how the parameter is documented on the Google API, but in Site Kit, `containerID` is actually the container's `publicId` (the ID used in the tag). Here `container_id` is what we refer to as the `internalContainerID`. We should use the same name for consistency.

Reply:
Updated.